src/plat/pc99/machine/intel-vtd.c

```diff
@@ -265,6 +278,33 @@ void vtd_handle_fault(void)
     }
 }
 
+void vtd_create_irte(word_t interrupt_index, word_t vector, word_t pci_bus, word_t pci_dev, word_t pci_func)
```

Comment:
These two new functions should be behind an `#ifdef CONFIG_IOMMU`, otherwise the build fails with:

>  /github/workspace/kernel/src/plat/pc99/machine/intel-vtd.c:291:5: error: unknown type name 'vtd_irte_t'; did you mean 'vtd_rte_t'?

Reply:
Did you miss this comment, @abrandnewusername ?

Reply:
I forgot to add the IRTE structure for 32 bit. Fixed now.